src/main/java/org/folio/rest/impl/InitAPIImpl.java

```diff
@@ -24,13 +31,34 @@ public void init(Vertx vertx, Context context, Handler<AsyncResult<Boolean>> han
       result -> {
         if (result.succeeded()) {
           registerProxies(context);
+          initializeSystemProperties(handler);
           handler.handle(Future.succeededFuture(true));
         } else {
           handler.handle(Future.failedFuture(result.cause()));
         }
       });
   }
 
+  private void initializeSystemProperties(Handler<AsyncResult<Boolean>> handler) {
```

Comment:
Does the module start up If there is no property we expect?

Reply:
Yes, it will start up. Let's not block the start up atleast for this release, because there may be someone who is not hosted on AWS. Atleast they will be able to install the module , however it will not be usable